src/lib/components/submit/SubmitReview.svelte

```diff
@@ -48,11 +66,11 @@
 
 {#if nonNullish(neuronId) && nonNullish(content)}
 	<form on:submit|preventDefault={onSubmit}>
-		<h1 class="font-bold capitalize text-4xl md:text-6xl mb-12">
+		<h1 class="mb-12 text-4xl font-bold capitalize md:text-6xl">
```

Comment:
Why CSS changes?

Reply:
Sorry, I reverted this.

(I have a plugin installed that sorts classes, it's called "headwind". I disabled it now 😅)